main.go

```diff
@@ -6,31 +6,38 @@ import (
 	"os"
 	"strings"
 
+	"github.com/bitrise-io/go-steputils/stepconf"
 	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/cache"
 	"github.com/kballard/go-shellquote"
 )
 
+const (
+	cacheInputNone = "none"
+	cacheInputAll  = "all"
+)
+
 // ConfigsModel ...
 type ConfigsModel struct {
```

Comment:
please rename this struct not to be exported, and also because in the log it will be visible like: 
```
ConfigsModel:
- input: value
....
```

Reply:
Renamed.